src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -98,6 +100,8 @@
         this.samlRolesSeparator = settings.get("roles_seperator");
         this.kibanaRootUrl = settings.get("kibana_url");
 
+        this.samlRolesPattern = Pattern.compile(samlRolesSeparator);
```

Comment:
it is not roles pattern, it is roles **separator** pattern.

Reply:
Done.